src/test/java/reposense/parser/AuthorConfigParserTest.java

```diff
@@ -183,4 +185,32 @@ public void parse_multipleColumnsWithCommasAndDoubleQuotes_success() throws Exce
             Assert.assertEquals(AUTHOR_ALIAS_COMMAS_AND_DOUBLE_QUOTES_MAP.get(author), author.getAuthorAliases());
         });
     }
+
+    @Test
+    public void parse_multipleRepos_success() throws Exception {
```

Comment:
I assume this tests the case where multiple repo locations are in a single row.

If so, make the test name prominent such that we know multiple locations in the same row are supposed to be captured

Reply:
Thanks for the suggestion. Done.